src/sentiment.js

```diff
@@ -0,0 +1,46 @@
+if (process.env.NODE_ENV !== 'production') {
+  /* eslint-disable global-require */
+  require('dotenv').config();
+}
+
+const Language = require('@google-cloud/language');
+const { fetchMessageBatch, addSentimentScore } = require('../repositories/sentiment-repository');
+
+const language = Language();
+
+
+function analyzeSentimentOfText(messageString, channelId, numberOfMessages) {
+  const document = {
+    content: messageString,
+    type: 'PLAIN_TEXT',
+  };
+
+  return language.analyzeSentiment({ document })
+    .then((results) => {
+      const sentimentScore = results[0].documentSentiment.score;
+      const magnitudeScore = results[0].documentSentiment.magnitude;
+      return addSentimentScore(
+        sentimentScore,
+        magnitudeScore,
+        channelId,
+        numberOfMessages,
+      );
+    })
+    .then(scoreData => console.log(scoreData[0].score))
```

Comment:
@meghanprestemon please confirm this `console.log` is required, and if so, disable it from being flagged by the linter, thanks~